### PR TITLE
Create Buttons

### LINKED ISSUE #4: Create Buttons
### User Story
As a user and as a developer I need buttons! I will build them in storybook, so i can use the buttons with different stylings on different sites. 

### Clarification

To do
Create buttons for: 
- [ ] sign in
- [ ] log in
- [ ] save
- [ ] delete
- [ ] new data
- [ ] new child
- [ ] new parent
- [ ] nav bar buttons / active

### Criteria
- [ ] Use design prototype
- [ ] Use storybook to build the components
- [ ] different sizes
- [ ] different colours
- [ ] different borders

### Material | Images
Design:
https://xd.adobe.com/view/7f86dc69-703e-48db-70d4-8d3d07276926-09b7/
Button examples:
image

image

image

### PR DESCRIPTION
**Pull request by meloneminze**
I created two different button types and stories in storybook. 
The XD Template: https://xd.adobe.com/view/7f86dc69-703e-48db-70d4-8d3d07276926-09b7/
closes #4

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -9,6 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
```
Comment (Lenkly) [inline client/public/index.html:15]: this could belong to another pull request :)

File: client/public/index.html [hunk 2]
```diff
@@ -9,6 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
```
Comment (Johannafendrich) [inline client/public/index.html:10]: You can enter the name of your app as title and a short description as content ;)

File: client/public/index.html (hunk 1 above)
Comment (meloneminze) [reply client/public/index.html:15]: Thank you. I know 🙈

File: client/public/index.html (hunk 2 above)
Comment (meloneminze) [reply client/public/index.html:10]: Thank you, i missed to change this.
